en/docs/getting-started/about-this-release.md

```diff
@@ -62,13 +62,18 @@ WSO2 API Microgateway 3.2.0 is the successor of WSO2 API Microgateway version 3.
     WSO2 API Microgateway when it is integrated with WSO2 API Manager.   
     <!---TODO: add documentation and point from here--->
 
--   **Integrate with Third Party Key Managers**     
+-   **Integrate with multiple Third Party Key Managers**     
     WSO2 API Microgateway 3.2.0,  is able to integrate with other key manager including the WSO2 API Manager Key Manager.
     
 -   **JWT Revocation with WSO2 API Manager**    
     WSO2 API Microgateway has ability to persist revoked access tokens inside WSO2 API Manager (Traffic Manager). 
     For more information, visit [JWT Token Revocation](how-tos/security/jwt-token-revocation/#jwt-token-revocation).
     
+-   **Support reading open API definition in interceptors**
+   
+-   **Support reading configurations from java interceptors**
+
+-   **Mutual SSL support per API**    
```

Comment:
Shall we add a small description of what was improved? mutual SSL per API support was there in 3.1.0 as well.

Reply:
yes.. we need the descriptions from the team. Please go ahead and add them.

Reply:
@praminda @VirajSalaka @menakaj Would you be able to help in adding a description?

Reply:
```suggestion
-   **Mutual SSL support per API**     
    Provides ability to specify which certificates should be trusted by each API. Prior to this improvement, any of the microgateway's trusted certificates could be used to invoke a mutual SSL protected API.
```